src/js/AddFormUI.js

```diff
@@ -40,16 +42,19 @@ class AddFormUI {
       this.addBtn.style.display = 'block';
     });
 
-    this.confirmBtn.addEventListener('click', () => {
-      const newCard = {
-        title: this.input.value,
-        description: '',
-        status: this.status,
-      };
+    this.confirmBtn.addEventListener('click', async () => {
+      if (this.input.value.trim()) {
+        const newCard = {
+          title: this.input.value,
+          description: '',
+          status: this.status,
+        };
 
-      this.input.value = '';
+        this.input.value = '';
 
-      CardAPI.createCard(newCard);
+        const card = await CardAPI.createCard(newCard);
+        new CardUI(card).renderCard(target.querySelector('.card-list'));
```

Comment:
```suggestion
        CardAPI.createCard(newCard)
          .then(card => {
            new CardUI(card).renderCard(target.querySelector('.card-list'));
          });
```

i think this variant is better

Reply:
Why you think so, any arguments?

Reply:
we will assign a value to the `card` variable only when the response returns and only then the next function will be called.
I think it is unnecessary to write the response to the variable if it is available in the `then` construction

Reply:
> we will assign a value to the `card` variable only when the response returns and only then the next function will be called.
> I think it is unnecessary to write the response to the variable if it is available in the `then` construction

It make sense, but not critical

Reply:
resloved in 9fb7e67